test/TEST-75-RESOLVED/test.sh

```diff
@@ -17,6 +17,12 @@ if ! knotc -c "${TEST_BASE_DIR:?}/knot-data/knot.conf" conf-check; then
     exit 0
 fi
 
+# We need nftables to test the serve stale feature
+if ! nft -V; then
```

Comment:
Rather than doing this, please check around the installation bit below, and then check again at runtime and skip the new test body only. That way we don't lose coverage entirely just because nft is missing/unusable.

Reply:
also, the way we check for these is `if command -v nft >/dev/null; then` as it might be present but not runnable for any number of reasons